spaces/S000014/properties/P000126.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S000014
+property: P000126
+value: true
+---
+
+For every $U\subset X$, either $0\in U$ so $U$ is closed or $0\not\in U$ so $U$ is open.
```

Comment:
```suggestion
For every $A\subseteq X$, if $0\not\in A$ then $A$ is open. Otherwise, $0\in A$ so $(-1,1)\not\subseteq X\setminus A$, and likewise $0\not\in X\setminus A$; therefore $X\setminus A$ is open and $A$ is closed.
```